ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/execution/ExecutionPayloadResult.java

```diff
@@ -51,7 +51,7 @@ public Optional<SafeFuture<HeaderWithFallbackData>> getExecutionPayloadHeaderFut
     return executionPayloadHeaderFuture;
   }
 
-  public Optional<SafeFuture<BlobsBundle>> getBlobsBundleFuture() {
+  public Optional<SafeFuture<BlobsBundle>> getBlobsBundle() {
```

Comment:
I think we need some consistency here, in this exact class. I'm ok with both versions, all future or all w/o future including vars

Reply:
thanks, i changed them all to future suffix